connector/spanmetricsconnector/README.md

```diff
@@ -1,16 +1,22 @@
 # Span Metrics Connector
 
-| Status                   |                                                           |
-| ------------------------ | --------------------------------------------------------- |
-| Stability                | [alpha]                                                   |
-| Supported pipeline types | See [Supported Pipeline Types](#supported-pipeline-types) |
-| Distributions            | [contrib], [sumo]                                         |
+<!-- status autogenerated section -->
+| Status        |           |
+| ------------- |-----------|
+| Distributions | [contrib], [sumo] |
 
+[alpha]: https://github.com/open-telemetry/opentelemetry-collector#alpha
+[contrib]: https://github.com/open-telemetry/opentelemetry-collector-releases/tree/main/distributions/otelcol-contrib
+[sumo]: https://github.com/SumoLogic/sumologic-otel-collector
 ## Supported Pipeline Types
```

Comment:
@mwear do you have a chance to follow up on this and add a line break in the mdatagen template?

Reply:
👍  i'll take care of that tomorrow